streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/AppBuildPipeServiceImpl.java

```diff
@@ -445,16 +445,13 @@ public Map<Long, PipelineStatus> listPipelineStatus(List<Long> appIds) {
             return Maps.newHashMap();
         }
         LambdaQueryWrapper<AppBuildPipeline> queryWrapper = new LambdaQueryWrapper<AppBuildPipeline>()
-            .select(AppBuildPipeline::getAppId, AppBuildPipeline::getPipeStatusCode)
             .in(AppBuildPipeline::getAppId, appIds);
-        List<Map<String, Object>> rMaps = baseMapper.selectMaps(queryWrapper);
+
+        List<AppBuildPipeline> rMaps = baseMapper.selectList(queryWrapper);
```

Comment:
Should the name of the variable `rMaps` be changed?

Reply:
> @wolfboys Except that the naming of the variable `rMaps` needs to be standardized, I have not found any other problems. I will test it as a whole after the merger

thanks for your review. I will improve what you mentioned